src/botcity/maestro/interfaces/common.ts

```diff
@@ -12,3 +12,9 @@ export interface Pageable {
   paged: boolean
   unpaged: boolean
 }
+
+export interface IColumn {
```

Comment:
What is the difference between `IColumn` and `Column`?

Reply:
It's more to implement the interface in the class.

It's not necessary in this case, but it can help with interacting with the IDE.